app/controllers/concerns/hal_actions.rb

```diff
@@ -59,11 +59,10 @@ def zoom_param
     }.call
   end
 
-
   def index_cache_path
     index_resources = self.try(:resources_base) || resources
     PagedCollection.new(
-      with_params(self.class.resources_params, index_resources),
+      decorate_query(index_resources),
```

Comment:
Trying to get a handle on this, is [this bit](https://github.com/PRX/cms.prx.org/blob/71164ab63b3a78f93c9a26eb311cfaaaa15e7913/app/models/paged_collection.rb#L27) the thing that protects us from generating the same cache key for multiple collections containing the same most recently updated thing but different constituents? I suspect this becomes a greater concern when we start allowing the `per` parameter, so it is probably something to keep an eye on.